Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -45,6 +45,30 @@ public extension Array {
             return thisIndex < otherIndex
         }
     }
+    
+    /**
+     SwifterSwift: Adds a new element at the end of the array, mutates the array in place
+     - Parameter newElements: The optional element to append to the array
+     */
+    @discardableResult
+    mutating func append(_ newElement: Self.Element?) -> [Element]
```

Comment:
I think we wouldn't need to return the array here... just. avoid mutating func would be fine...